README.md

```diff
@@ -74,3 +64,4 @@ If you changed the ABI of a module or `GenericVotesTimestampToken.sol` you'll wa
 - [Vercel](https://vercel.com/?utm_source=jokedao&utm_campaign=oss).
 
 JokeDAO V2 front-end is hosted on [Vercel](https://vercel.com/?utm_source=jokedao&utm_campaign=oss).
+JokeDAO V2 front-end is hosted on [Vercel](https://vercel.com/?utm_source=jokedao&utm_campaign=oss).
```

Comment:
Probably this was doubled unintentionally :)

Reply:
Oo good call, thank you!